whisper-finetune/inference_endpoint/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ptebic.azurecr.io/internal/azureml/aifx/nightly-ubuntu2004-cu117-py38-torch210dev:20230505
```

Comment:
Only nightly ACPT is working with Whisper? Since internal packages will be removed in 3 months, it'd be better to use public or MCR images.

Reply:
Yes, we require ORT nightly or ORT 1.15 once it releases.

Reply:
I will update this once a public/MCR image is available with ORT 1.15